manifests/init.pp

```diff
@@ -10,6 +10,10 @@
 #   number of days to keep the backup history
 # @param cleanup_empty_backup_dirs
 #   enable/disable a post command to cleanup empty directories with $destination. They can occour if the system run out of free diskspace
+# @param dependencies
+#   array of dependencies for the database dump script
+# @param manage_dependencies
+#   Boolean to enable/disable the installation of the dependencies. defaults to false because the parameter was added after the initial release
```

Comment:
This reads the default as `false` though the code sets it to `true`.

Reply:
ah good catch, fixed